src/plugins/Trader/types.ts

```diff
@@ -2,7 +2,7 @@ export interface Settings {
     currency: Currency
 }
 
-export enum Platform {
+export const enum Platform {
```

Comment:
Please revert this. `const enum` is conflict with `transpileModule`

Reply:
why?

Reply:
Ok, only `const enum` in the ambient module context is not allowed. https://github.com/microsoft/TypeScript/issues/20703